akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/Effect.scala

```diff
@@ -10,8 +10,8 @@ import akka.annotation.DoNotInherit
  * All tracked effects for the [[akka.actor.testkit.typed.scaladsl.BehaviorTestKit]] and
  * [[akka.actor.testkit.typed.javadsl.BehaviorTestKit]] must extend this type.
  *
- * Factories/types for effects are available through [[akka.actor.testkit.typed.scaladsl.Effects]]
- * and [[akka.actor.testkit.typed.javadsl.Effects]]
+ * Factories/types for effects are available through [[akka.actor.testkit.typed.Effects]]
+ * and [[akka.actor.testkit.typed.javadsl.EffectFactory]]
```

Comment:
A bit inconsistent with the factories for Java in `javadsl` but the Scala side in the root package, wouldn't it be nicer to just mirror the naming in the `scaladsl`?

Reply:
+1 to create methods

Reply:
This gets hairy as scala can put the static methods on the MODULE$ member. I'll mirror the factories as originally suggested